codegen/bin/cointests

```diff
@@ -36,6 +36,28 @@ def self.coin_type(path)
   path.split('/')[2].chomp("'")
 end
 
+# Explorer urls
+def self.explorer_tx_url(c)
+  path = c['explorer']['url'].to_s + c['explorer']['txPath'].to_s
+end
+def self.explorer_account_url(c)
+  path = c['explorer']['url'].to_s + c['explorer']['accountPath'].to_s
+end
+def self.explorer_sample_tx(c)
+  if c['explorer']['sampleTx'].nil?
+    "t123"
+  else
+    c['explorer']['sampleTx']
+  end
+end
```

Comment:
do we need `sampleTx ` and `sampleAccount` ? I would rather keep it simpler

Reply:
This is for test only.  @hewigovens  proposed to use real IDs in coinTypeTests (not '123'), which has the advantage that opening the url shows a real page.  But since the tests are generated too, the only way not to override these value if they are in the coins.json.  They are optional, don't add to clutter where they are missing.  But I agree that they have limited value.